src/test/java/com/inventorsoft/junit/mapper/BookMapperTest.java

```diff
@@ -0,0 +1,149 @@
+    static final String TITLE = "Mr.Mercedes";
+    static final LocalDate RELEASE_DATE = LocalDate.of(2014, 6, 3);
+    static final String DESCRIPTION = "Novel";
+
+    static final Long AUTHOR_ID = 2L;
+    static final String AUTHOR_NAME = "Stephen King";
+    static final LocalDate AUTHOR_BIRTHDAY = LocalDate.of(1947, 9, 21);
+    static Author author = new Author();
+    static Book book = new Book();
+    static BookResponse bookResponse = new BookResponse();
+    static AuthorResponse authorResponse = new AuthorResponse();
+
+    @BeforeEach
+    void init() {
+        author.setBirthday(AUTHOR_BIRTHDAY);
+        author.setName(AUTHOR_NAME);
+        author.setId(AUTHOR_ID);
+
+        book.setAuthor(author);
+        book.setDescription(DESCRIPTION);
+        book.setReleaseDate(RELEASE_DATE);
+        book.setId(ID);
+        book.setTitle(TITLE);
+
+        authorResponse.setBirthDate(AUTHOR_BIRTHDAY);
+        authorResponse.setName(AUTHOR_NAME);
+        authorResponse.setId(AUTHOR_ID);
+
+        bookResponse.setId(ID);
+        bookResponse.setTitle(TITLE);
+        bookResponse.setReleaseDate(RELEASE_DATE);
+        bookResponse.setDescription(DESCRIPTION);
+        bookResponse.setAuthor(authorResponse);
+    }
+
+
+    @Test
+    void mapEntitiesToResponses() {
+        List<Book> list = List.of(book);
+        List<BookResponse> expectedResult = List.of(bookResponse);
+        when(authorMapper.mapEntityToResponse(book.getAuthor())).thenReturn(authorResponse);
+        List<BookResponse> actualResult = bookMapper.mapEntitiesToResponses(list);
+
+
+        assertEquals(expectedResult.size(), actualResult.size());
+        for (int i = 0; i < actualResult.size() && i < expectedResult.size(); i++) {
+            assertEquals(expectedResult.get(i).getId(), actualResult.get(i).getId());
+            assertEquals(expectedResult.get(i).getTitle(), actualResult.get(i).getTitle());
+            assertEquals(expectedResult.get(i).getReleaseDate(), actualResult.get(i).getReleaseDate());
+            assertEquals(expectedResult.get(i).getDescription(), actualResult.get(i).getDescription());
+            assertEquals(expectedResult.get(i).getAuthor(), actualResult.get(i).getAuthor());
+        }
+    }
+
+    @Test
+    void mapEntityToResponse() {
+        when(authorMapper.mapEntityToResponse(book.getAuthor())).thenReturn(authorResponse);
+
+        BookResponse actualResponse = bookMapper.mapEntityToResponse(book);
+
+        assertEquals(bookResponse.getId(), actualResponse.getId());
+        assertEquals(bookResponse.getTitle(), actualResponse.getTitle());
+        assertEquals(bookResponse.getDescription(), actualResponse.getDescription());
+        assertEquals(bookResponse.getReleaseDate(), actualResponse.getReleaseDate());
+        assertEquals(bookResponse.getAuthor(), actualResponse.getAuthor());
+    }
+
+    @Test
+    void mapCreateRequestToEntity() {
+        CreateBookRequest br = new CreateBookRequest();
```

Comment:
I would recommend not using shortcuts for variable names
```suggestion
        CreateBookRequest bookRequest = new CreateBookRequest();
```